rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1272,10 +1272,6 @@ def _create_sequence(
             transformer_inputs, 1 - mask, self._training
         )
 
-        if self.config[NUM_TRANSFORMER_LAYERS] > 0:
-            # apply activation
-            outputs = tfa.activations.gelu(outputs)
-
```

Comment:
We need to keep this if we want to reapply the changes again.

Reply:
Thanks, I thought so. Not sure what went wrong, i'll try again.

Reply:
Wait sorry i should clarify -- this PR was supposed to already reapply the changes. Is it possible that the activation PR wasn't ever merged into master (but genie's were)?

Reply:
Would be strange... but maybe there were some merge conflicts and someone did not resolve them properly... 
Just checked master. The changes over there are correct.

Reply:
Okay. I've tried a few different things with the commits and haven't gotten it to play nice. 5565 is correct on master and no diff, and the right side here looks correct (even if the diff doesn't correctly reflect master). What if we merge, then check master to make sure everything is how we expect? I _think_ the end result of this PR is correct...

Reply:
Can't we just change it manually? But sure, we can also merge and see what happens.

Reply:
I mean, the goal of this PR was to re-apply the changes, and in this comment for example the change is applied, the activation has been moved to the transformer block as expected

Reply:
but it should not be moved to the transformer block, it should stay here

Reply:
ok now i'm offically confused because [Move activation to transformer block](https://github.com/RasaHQ/rasa/pull/5626/files) the PR i'm trying to re-apply. but it seems that that pr, despite the name, did not move the activation to the transformer block?

Reply:
it may make more sense for you to take over this merge/re-apply since i don't know intuitively which changes are supposed to go where 😬

Reply:
Yeah, sorry. We first moved it but then decided to keep it where it was and wrap it with an if statement.

Reply:
ok let's definitely rename that pr 🙈 i think i missed a few commits, let me try

Reply:
Updated everything. Looks good now I would say.

Reply:
Great, thank you! I think i just missed 2 of your commits which made stuff weird. renamed the PR.